sansa-owl-spark/pom.xml

```diff
@@ -15,19 +15,88 @@
     <name>SANSA OWL layer - SPARK</name>
     <description>A library to read OWL files into Spark.</description>
 
+    <properties>
+    </properties>
+
     <dependencies>
         <dependency>
             <groupId>net.sansa-stack</groupId>
             <artifactId>sansa-owl-common_${scala.binary.version}</artifactId>
             <version>${project.version}</version>
         </dependency>
 
+
+
+        <!-- To remove warnings from xerec parser -->
+        <dependency>
+            <groupId>xalan</groupId>
+            <artifactId>xalan</artifactId>
+            <version>2.7.2</version>
```

Comment:
This snippet was seemingly copied from a 2017 StackOverflow answer. The 2.7.2 release of the xalan:xalan package stems from the year 2014. To me it is not clear why exactly this dependency is needed here.